### DISCUSSION TITLE
想问下沉浸式翻译是默认用浏览器自带同步来同步配置的吗

### DISCUSSION THREAD

**Discussion by calmwaves111** [immersive-translate/immersive-translate#796] [Help]
### 概述 | Summary

我用的是edge浏览器，前两天重装了系统，重新在edge上登录微软账号之后，利用同步功能直接把所有插件同步了过来，发现沉浸式翻译的配置仍然是我之前的配置而没有重置，所以想问问

**Comment by theowenyoung** [comment]
是的。
